release/ray_release/test.py

```diff
@@ -128,6 +128,12 @@ def from_bazel_event(cls, event: dict, team: str):
             }
         )
 
+    @classmethod
+    def gen_from_name(cls, name: str):
+        s3_prefix = name.replace("/", "_")
```

Comment:
why do we need this name replace? is this replace pattern also used elsewhere?

Reply:
the `/` character is not allowed as an aws s3 key

Reply:
and good catch, i can just reuse this function https://github.com/ray-project/ray/blob/master/release/ray_release/test.py#L228